presto-docs/src/main/sphinx/sql/select.rst

```diff
@@ -78,7 +87,7 @@ GROUP BY Clause
 ---------------
 
 The ``GROUP BY`` clause divides the output of a ``SELECT`` statement into
-groups of rows containing matching values. A ``GROUP BY`` clause may
+groups of rows containing matching values. A simple ``GROUP BY`` clause may 
```

Comment:
Looks like this added a trailing space? I also see these on other lines. Make sure your editor is stripping them on save